ccmlib/common.py

```diff
@@ -591,3 +602,26 @@ def assert_jdk_valid_for_cassandra_version(cassandra_version):
     if cassandra_version >= '3.0' and get_jdk_version() < '1.8':
         print_('ERROR: Cassandra 3.0+ requires Java >= 1.8, found Java {}'.format(get_jdk_version()))
         exit(1)
+
+
+def merge_configuration(original, changes, delete_empty=True):
```

Comment:
This is recursive, so could blow stack with arbitrarily deep config, but figure that is ok as I wouldn't expect dse/cassandra yaml to be very nested.